src/main/java/edu/harvard/iq/dataverse/ingest/tabulardata/impl/plugins/dta/DTAFileReader.java

```diff
@@ -328,9 +312,9 @@
         "time", "date", "date", "date", "date", "date", "date",
         "date", "date", "date", "date", "date", "date"
     };
-    private static Map<String, String> DATE_TIME_FORMAT_TABLE=  new LinkedHashMap<String, String>();
+    private static Map<String, String> DATE_TIME_FORMAT_TABLE=  new LinkedHashMap<>();
 
-    private static long SECONDS_PER_YEAR = 24*60*60*1000L;
+    private static long SECONDS_PER_YEAR = 525600L;
```

Comment:
This is a functional change, but there are not 1000 days per year, so I'm really unsure how the old could have been correct